media/js/firefox/onboarding/fxa-complex.js

```diff
@@ -0,0 +1,102 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function($) {
```

Comment:
By no means a blocker for this test, but it feels like we're starting to repeat most of the code in this file for each new page that uses FxA. 

Think it's worth filing a future bug to try and consolidate this in a standard base library that we can re-use?